Makefile

```diff
@@ -36,3 +36,9 @@ clean_all_service:
 
 clean_all_tools:
 	cd tools && make clean
+
+add_service:
```

Comment:
I'm starting to think this makefile structure we have is getting unmanageable. They're odd targets too. What do you think?